server/global/data_importer/dataImporter.js

```diff
@@ -4,6 +4,13 @@ import Logger from "../Logger.js";
 
 import { parseMoleculesFromCsv } from "../molecules_parser/Parser.js";
 
+export const MAX_LENGTH = {
```

Comment:
```suggestion
export const MAX_LENGTHS = {
```
(watch out to rename in an IDE :smile:)